lib/src/checkbox/checkbox.component.scss

```diff
@@ -33,6 +33,36 @@
         }
     }
 
+    .hc-checkbox-indeterminate {
```

Comment:
Added indeterminate overlay here

Reply:
I really like it! Took me a google to figure out what indeterminate meant

Reply:
Wow, this looks very nice! Great job